src/app/content/utils/urlUtils.ts

```diff
@@ -40,6 +44,29 @@ export const getUrlParamsForBook = (
   }
 };
 
+export const getContentParams = (appState: AppState) => {
```

Comment:
yea if we're deriving these on the fly it'd be nicer to just save the ones that got dispatched in the first place. again, you might not need to do that if you use `pop`, you may end up just removing this